src/pages/Home.jsx

```diff
@@ -51,9 +51,10 @@ function Home() {
   }, [countries, searchInput, selectedRegion]);
 
   useEffect(() => {
-    fetchDate();
-    setTimeout(() => setLoading(true), 1000);
-  }, [fetchDate]);
+    if (countries.length) setLoading(true);
+    fetchDate(render);
+    render.current++;
```

Comment:
Slava. Who taught you this?

Reply:
what is wrong?
if we received then isLoading true, if not false

Reply:
I think it's very elegant :)

Reply:
I'm not saying about loading. It's about this `render.current++;`

Reply:
so it’s even more elegant, we get a counter, as soon as it’s rendered once we won’t load any more data

Reply:
So you're saying - if I already have countries, let's set loading to true and load them. 

And let's pass render object to a place from where I took it and even after end of the request let's increment useless number of renders.

And you're saying it's elegant, right?

Reply:
Exactly, why am I calling fetchData if I can check the quantity of render here?

Reply:
Maybe it was a bad idea to communicate in English.

When I asked you to count number of render, I meant console log it, not store in the app.